storage/pruning/pruningStorer.go

```diff
@@ -466,14 +476,13 @@ func (ps *PruningStorer) closeAndDestroyPersisters(epoch uint32) error {
 	return nil
 }
 
-// getNewFilePath will return the file path for the new epoch. It uses regex to change the default path
-func (ps *PruningStorer) getNewFilePath(epoch uint32) string {
-	// TODO: the path will be provided by a path naming component
-	// using a regex to match the epoch directory name as placeholder followed by at least one digit
-	// in a string which contains Epoch_X it will replace X with the given epoch number
-	rg := regexp.MustCompile(`Epoch_\d+`)
-	newEpochDirectoryName := fmt.Sprintf("%s_%d", DefaultEpochDirectoryName, epoch)
-	return rg.ReplaceAllString(ps.dbPath, newEpochDirectoryName)
+func getShardIdString(coordinator sharding.Coordinator) string {
```

Comment:
This code is duplicated in pruningStorerFactory. It could be hard to debug if for example the shardIdStr from "metachain" would be changed to uint32.MaxInt or something else.

Reply:
done. extracted to a function in core

Reply:
ok